script/DeployJSONExtensionRegistry.s.sol

```diff
@@ -12,9 +12,26 @@ contract DeployRegistry is ScriptBase {
16836001600160a01b03167f032e3f01605d64bb10db2ed841df91cefce984315da40cfd2e3c001011925f1c846040516102539190610476565b60405180910390a3505050565b6001600160a01b03811660009081526020819052604090208054606091906102879061060c565b80601f01602080910402602001604051908101604052809291908181526020018280546102b39061060c565b80156103005780601f106102d557610100808354040283529160200191610300565b820191906000526020600020905b8154815290600101906020018083116102e357829003601f168201915b50505050509050919050565b6000816001600160a01b0316836001600160a01b03160361032f5750600161018c565b6001600160a01b0383163b1561044357826001600160a01b0316638da5cb5b6040518163ffffffff1660e01b8152600401602060405180830381865afa925050508015610399575060408051601f3d908101601f1916820190925261039691810190610755565b60015b156103c357826001600160a01b0316816001600160a01b0316036103c157600191505061018c565b505b604051632474521560e21b8152600060048201526001600160a01b0383811660248301528416906391d1485490604401602060405180830381865afa92505050801561042c575060408051601f3d908101601f1916820190925261042991810190610772565b60015b1561044357801561044157600191505061018c565b505b50600092915050565b60006020828403121561045e57600080fd5b81356001600160e01b0319811681146101be57600080fd5b600060208083528351808285015260005b818110156104a357858101830151858201604001528201610487565b506000604082860101526040601f19601f8301168501019250505092915050565b6001600160a01b03811681146104d957600080fd5b50565b600080604083850312156104ef57600080fd5b82356104fa816104c4565b9150602083013561050a816104c4565b809150509250929050565b634e487b7160e01b600052604160045260246000fd5b6000806040838503121561053e57600080fd5b8235610549816104c4565b9150602083013567ffffffffffffffff8082111561056657600080fd5b818501915085601f83011261057a57600080fd5b81358181111561058c5761058c610515565b604051601f8201601f19908116603f011681019083821181831017156105b4576105b4610515565b816040528281528860208487010111156105cd57600080fd5b8260208601602083013760006020848301015280955050505050509250929050565b60006020828403121561060157600080fd5b81356101be816104c4565b600181811c9082168061062057607f821691505b60208210810361064057634e487b7160e01b600052602260045260246000fd5b50919050565b601f82111561069057600081815260208120601f850160051c8101602086101561066d5750805b601f850160051c820191505b8181101561068c57828155600101610679565b5050505b505050565b815167ffffffffffffffff8111156106af576106af610515565b6106c3816106bd845461060c565b84610646565b602080601f8311600181146106f857600084156106e05750858301515b600019600386901b1c1916600185901b17855561068c565b600085815260208120601f198616915b8281101561072757888601518255948401946001909101908401610708565b50858210156107455787850151600019600388901b60f8161c191681555b5050505050600190811b01905550565b60006020828403121561076757600080fd5b81516101be816104c4565b60006020828403121561078457600080fd5b815180151581146101be57600080fdfe68747470733a2f2f646f63732e7a6f72612e636f2f6a736f6e2d636f6e74726163742d7265676973747279a2646970667358221220d5323d1bd4700d180f8289d3926fae918f6d4f39522a6a9e9daa7a0a1755af6f64736f6c63430008110033";
```

Comment:
minor suggestion: storing this code in an external json, along with the salt and expected address.